template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/RazorPagesWeb-CSharp/Pages/Account/Manage/Index.cshtml.cs

```diff
@@ -53,18 +53,19 @@ public class InputModel
 
         public async Task<IActionResult> OnGetAsync()
         {
-            var user = await _userManager.GetUserAsync(HttpContext.User);
+            var user = await _userManager.GetUserAsync(User);
             if (user == null)
             {
                 return RedirectToPage("/Error");
             }
 
-            Username = user.UserName;
+            Username = await _userManager.GetUserNameAsync(user);
```

Comment:
Why do we need username on the model?  This should already be in the ClaimsPrincipal (User.Identity.Name) for the manage page, we aren't allowing this to be changed are we?